### DISCUSSION TITLE
Connection to "pos.domaine.com:25" has been closed unexpectedly.

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2882] [Help with using Postal]
Hi, 

I use mailwizz with postal servers and I have a recurring error in the sending logs:

`Connection to "pos.domaine.com:25" has been closed unexpectedly.`

A simple restart can restart the server, but I wanted to know if I can resolve the problem in a more lasting way.

Thx

**Comment by willpower232** [comment]
When you say a simple restart, are you restarting your whole server or just the Postal bit? Can you see any errors before then?

**Reply by SaasX** [reply]
Restarting server

**Reply by willpower232** [reply]
Did you check the status of Postal before restarting?

**Reply by SaasX** [reply]
Hi, 
I'm keeping this post as a favorite, and I'll give you additional information as soon as I have the problem again.

**Reply by SaasX** [reply]
Hi,

It seems to be related to a disk space issue.

The disk is 100% full, and 50% after reboot.

It's a small SMTP, few emails are sent, so I'm surprised that 10GB of disk space fills up in a few days.

Do you have any thoughts in this direction?

Thx.

**Reply by willpower232** [reply]
10GB isn't that much space at all. If you aren't allowed any bigger then you would have to use something to verify where all the hard drive space had gone

**Reply by SaasX** [reply]
The total disk size is 20GB.

I have around 250 substation servers with this configuration, and this is the only one with a problem.

The 10GB I mention is the free space after reboot.

I don't understand what fills 10GB in 3 days when the server is dedicated to postal and sends few messages.

**Reply by willpower232** [reply]
Its possible you need to configure docker logs because this one is having lots of SMTP traffic or some other kind of activity which is filling it up.

**Reply by SaasX** [reply]
Ok, thanks for your feedback.

I'll look in that direction.

**Reply by adamcooke** [reply]
https://docs.postalserver.io/features/logging

**Reply by SaasX** [reply]
Filesystem      Size  Used Avail Use% Mounted on
tmpfs           192M  1.5M  190M   1% /run
/dev/sda1        20G   17G  2.8G  86% /
tmpfs           956M     0  956M   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
/dev/sda15      105M  6.1M   99M   6% /boot/efi
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/32f32fec1d935557960c3179fac4ba02c1e0ba4e1727df4b7d52820c3e3e06b2/merged
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/a94e9fcf5869b20c5ac8abd6ce24b2aa0804ebde83382cb1847a5241202e39a0/merged
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/036d9dc1108de2984647a102f57289481f9d636807bbf989d6491bdcbace2873/merged
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/886ff717de7581db08b7f712a0efec413b232f7886472e2815096e61b085774a/merged
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/dc9ab98e5460124b8444c11073e835c815d0f411eb5aad37dce32d403e2d092c/merged
overlay          20G   17G  2.8G  86% /var/lib/docker/overlay2/77ab12eefa4199f611de7ddddd2d9993494ba99cdd96b22b565fe761a27dff94/merged
tmpfs           192M  4.0K  192M   1% /run/user/1000

**Reply by SaasX** [reply]
I'll give you a status of the disk for information.
I have just set up the parameters of your links, I will keep you informed.
